src/main/scala/it/pagopa/interop/apigateway/api/impl/package.scala

```diff
@@ -168,10 +167,13 @@ package object impl extends SprayJsonSupport with DefaultJsonProtocol {
 
   implicit class EnrichedAgreementState(private val agreement: AgreementManagementApiAgreementState) extends AnyVal {
     def toModel: AgreementState = agreement match {
-      case AgreementManagementApiAgreementState.ACTIVE    => AgreementState.ACTIVE
-      case AgreementManagementApiAgreementState.INACTIVE  => AgreementState.INACTIVE
-      case AgreementManagementApiAgreementState.PENDING   => AgreementState.PENDING
-      case AgreementManagementApiAgreementState.SUSPENDED => AgreementState.SUSPENDED
+      case AgreementManagementApiAgreementState.DRAFT                        => AgreementState.DRAFT
```

Comment:
should this be exposed?

Reply:
@beetlecrunch ?

Reply:
No

Reply:
How am I supposed to hide it? I just made it recompile 😅

Reply:
descriptors state has the same behavior (returning an `Either`)

Reply:
Do I have to filter the draft events when listing? I have to 404 when I receive a punctual get on that specific agreement?

Reply:
A DRAFT agreement does not exist on the API side, so 404 looks fine.